web-components/src/components/form/CreditSendForm.tsx

```diff
@@ -129,7 +129,7 @@ const CreditSendForm: React.FC<FormProps> = ({
       initialValues={initialValues}
       validate={validateHandler}
       onSubmit={async values => {
-        track('send2');
+        track('send2'); //TODO: move to onSubmit, to keep web-components tracking-agnostic regen-network/regen-registry#1365
```

Comment:
Noticed this was breaking the storybook story. Made a task to clean up regen-network/regen-registry#1365